README.md

```diff
@@ -212,18 +325,18 @@ and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
   modified with a module (e.g. the `ini` module) then default to using the `template` module. This both makes
   it easier to see what is being configured and to add additional, more complex options in the future.
 * Avoid the use of `lineinfile` wherever that might be feasible. Try editing files directly using other built
-  in modules (e.g. `ini_file`) or reading and parsing. If you are modifying more than a tiny number of lines
-  or in a manner more than trivially complex, try leveraging the `template` module, instead. This will allow
-  the entire structure of the file to be seen by later users and maintainers.
+  in modules (e.g. `ini_file`, `blockinfile`, `xml`) or reading and parsing. If you are modifying more than a tiny number of lines or in a manner more than trivially complex, try leveraging the `template` module, instead. This will allow
+  the entire structure of the file to be seen by later users and maintainers. The use of `lineinfile` should include a comment with justification.
 * Limit use of the `copy` module to copying remote files and to uploading binary blobs. For all other file
   pushes, use the `template` module. Even if there is nothing in the file that is being templated at the
   current moment, having the file handled by the `template` module now makes adding that functionality much
   simpler than if the file is initially handled by the `copy` and then needs to be moved before it can be
   edited.
-* When using the `template` module, refrain from appending `.j2` to the file name. This alters the syntax
-  highlighting in most editors and will obscure the benefits of highlighting for the particular file type or
-  filename. Anything under the `templates` directory of a role is assumed to be treated as a Jinja 2 template,
-  so adding the `.j2` extension is redundant information that is not helpful.
```

Comment:
This change does not actually change anything, except for reformatting the paragraph and losing markup. IMO, it should be omitted.